app/serializers/albumPhoto.js

```diff
@@ -0,0 +1,9 @@
+exports.albumPhotoSerializer = albumPhoto => ({
```

Comment:
I think that this file is unnecessary because you can put all album mappers in a single file.

Reply:
Fixed it and also named `albumPhoto` to `photo` because I realized that `albums/:id/photo` responses the same object that `/photo`.